crates/rollup/src/celestia_rollup.rs

```diff
@@ -0,0 +1,109 @@
+#![deny(missing_docs)]
+//! StarterRollup provides a minimal self-contained rollup implementation
+
+use async_trait::async_trait;
+use sov_db::ledger_db::LedgerDB;
+
+use sov_modules_api::default_context::{DefaultContext, ZkDefaultContext};
+use sov_modules_api::Spec;
+use sov_modules_stf_blueprint::kernels::basic::BasicKernel;
+use sov_risc0_adapter::host::Risc0Host;
+use sov_rollup_interface::services::da::DaService;
+use sov_state::config::Config as StorageConfig;
+use sov_state::{DefaultStorageSpec, ZkStorage};
+use sov_state::storage_manager::ProverStorageManager;
+use sov_stf_runner::RollupConfig;
+use stf_starter::Runtime;
+use sov_celestia_adapter::{CelestiaConfig, CelestiaService};
+use sov_celestia_adapter::types::Namespace;
+use sov_celestia_adapter::verifier::{CelestiaSpec, CelestiaVerifier, RollupParams};
+
+/// The namespace for the rollup on Celestia. Must be kept in sync with the "rollup/src/lib.rs"
+const ROLLUP_NAMESPACE: Namespace = Namespace::const_v0(*b"sov-celest");
+
+/// Rollup with [`MockDaService`].
+pub struct StarterRollup {}
```

Comment:
Let's change the name of the struct to `CelestiaRollup` and update the comment above.